pages/services/beta-dcos-monitoring/0.4.3-beta/getting-started/index.md

```diff
@@ -24,9 +24,7 @@ Please follow these [instructions](https://docs.mesosphere.com/1.12/administerin
 
 # Install {{ model.techName }} service
 
-## Download the package
-
-Download the `.dcos` package of the {{ model.techName }} service from the [Mesosphere support site](https://support.mesosphere.com/s/downloads).
+The `dcos-monitoring` package is installable via the catalog, it does not require installation from the support portal.
```

Comment:
1. Please remove the steps on installing the package registry. It's not needed since the package is part of the catalog. This includes in the "install the service" section.
2. You don't need to mention the "it does not require installation from the support portal". You should just provide the instructions on installing it.
3. In the "install the service" section below, you do not need to upload a file. You just need the dcos package install {{ model.packageName }} --package-version=<VERSION>
4.